docs/api-reference/web/adapters/slope.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: slope-adapter
+hide_table_of_contents: false
+displayed_sidebar: apiReference
+---
+
+# Slope Adapter
+
+---
+
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+## [`@web3auth/slope-adapter`](https://npmjs.com/package/@web3auth/slope-adapter)
+
+Slope adapter allows you to connect with slope wallet. You can read more about slope wallet here.(https://docs.slope.app/).
+
+## Basic Details
+
+---
+
+#### Adapter Name: `slope`
+
+#### Package Name: [`@web3auth/slope-adapter`](https://npmjs.com/package/@web3auth/slope-adapter)
+
+#### authMode: `DAPP`
+
+#### chainNamespace: `SOLANA`
+
+#### Default: `YES`
+
+## Installation
+
+---
+
+<Tabs
+  defaultValue="npm"
+  values={[
+    { label: "npm", value: "npm" },
+    { label: "yarn", value: "yarn" },
+  ]}
+>
+
+<TabItem value="npm">
+
+```shell
+npm install --save @web3auth/slope-adapter
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```shell
+yarn add @web3auth/slope-adapter
+```
+
+</TabItem>
+
+</Tabs>
+
+## Arguments
+
+---
+
+<Tabs
+  defaultValue="table"
+  values={[
+    { label: "Table", value: "table" },
+    { label: "Interface", value: "interface" },
+  ]}
+>
+
+<TabItem value="table">
+
+| Parameter   | type              |
+| ----------- | ----------------- |
+| chainConfig | CustomChainConfig |
+
+</TabItem>
+
+<TabItem value="interface">
+
+```ts
+interface SlopeAdapterOptions {
+  chainConfig?: CustomChainConfig;
+}
+```
+
+</TabItem>
+
+</Tabs>
+
+### Custom Chain Config
+
+#### `chainConfig`
+
+:::caution
+
+While you can pass your `chainConfig`, it is not required since you can directly pass it over to the `Web3Auth`/ `Web3AuthCore` configuration while
```

Comment:
since or if?

Reply:
I was preferring that people just pass it to Web3Auth and not to the adapter. What do you recommend?

Reply:
yeah passing in web3auth is better but give them a warning that if they pass it in adapter it will override the Web3Auth constructor chainConfig.

Reply:
got it

Reply:
done